docs/getting_started/setup.md

```diff
@@ -98,6 +98,18 @@ make
 ```
 Running Cmake will create `crow_all.h` file and place it in the build directory.
 
+##Installing Crow
+
+if you wish to use crow globally without copying `crow_all.h` in your projects, you can install `crow` on your machine by using `make install` command instead of `make` after `cmake..`. Thus, the procedure will look like
```

Comment:
maybe put Crow instead of `crow`, also the "instead of `make` after `cmake ..`" part is kind of redundant since the whole procedure is there